README.md

```diff
@@ -213,6 +213,31 @@ type FrameMetadataResponse = {
 <br />
 <br />
 
+## Identity Kit 👤
+
+### OnchainName
+
+The `OnchainName` component primarily focuses on showcasing ENS names for given Ethereum addresses, and defaults to displaying a sliced version of the address when an ENS name isn't available.
+
+```tsx
+import { OnchainName } from '@coinbase/onchainkit';
+
+<OnchainName address="0x1234567890abcdef1234567890abcdef12345678" sliced={false} />;
```

Comment:
I think we can do a bit more here to showcase a real example, like have a small component using `<OnchainName>`

And have as comment how the address will result.

Reply:
I am thinking add png file with little screenshot of the result? or we can have a link to stackbliz that show case the components.

Reply:
Let's circle back on this one. We still miss the default Tailwind css here.